cms/jinja2/templates/pages/information_page.html

```diff
@@ -0,0 +1,27 @@
+{% extends "templates/base_page.html" %}
+{% from "components/related-content/_macro.njk" import onsRelatedContent %}
+
+{% block main %}
+    {% if page.summary %}
+        <p>{{ page.summary }}</p>
+    {% endif %}
+
+    <h5>Last Updated: {{ page.last_updated }}</h5>
```

Comment:
This should be wrapped in a conditional, as the last updated field is optional. Currently, this would output `None`.

Reply:
Done!